spec/features/studies/create_study_spec.rb

```diff
@@ -105,7 +105,7 @@
     end
 
     choose('Open (ENA)', allow_label_click: true)
-    expect(page).not_to have_content('HMDMC approval number')
+    expect(page).to have_content('HuMFre approval number')
```

Comment:
Weird that this was `.not_to` before and has to be changed to `.to`.  Are we sure we understand what the original test was doing?

Reply:
There are three data release strategies: Open (ENA), Managed (EGA), and Not Applicable (Contact Datasharing).
You choose one of them using radio buttons. Previously, the approval number field was only available for Managed (EGA) data release strategy (choosing radio button). Now the approval number is visible for all three options.

The original test was making sure that it is only visible for Managed option. The new test is making sure that it is visible for all three options.

<img width="841" alt="Screenshot 2022-11-23 at 13 08 37" src="https://user-images.githubusercontent.com/91894/203555019-27c23663-2ea8-438d-ab29-cc00f0949f53.png">

Reply:
Ah nice.  Perfect then!  Thanks for the extra information.